Description: Do not declare any packages in the setup metadata
Author: Peter Pentchev <pp@storpool.com>
Last-Update: 2023-03-10

--- a/setup.py
+++ b/setup.py
@@ -32,7 +32,7 @@
                       '.git/refs/remotes/origin/*']}
 
 modules = \
-['py']
+['contextlib_chdir']
 setup_kwargs = {
     'name': 'contextlib-chdir',
     'version': '1.0.2',
@@ -43,8 +43,6 @@
     'maintainer': 'None',
     'maintainer_email': 'None',
     'url': 'None',
-    'packages': packages,
-    'package_data': package_data,
     'py_modules': modules,
     'python_requires': '>=3.6,<4.0',
 }
